binr/radare2/radare2.c

```diff
@@ -658,7 +658,24 @@ int main(int argc, char **argv, char **envp) {
 		}
 	}
 	if (noStderr) {
-		close (2);
+		if (-1 == close (2)) {
+			eprintf ("Failed to close stderr");
+			return 1;
+		}
+		int new_stderr = open ("/dev/null", O_RDWR);
```

Comment:
that will not work on a system without /dev mounted, also, this is wrong for windows

Reply:
but it's a good fix for that issue